<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="no_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Sayı 1: </string>
    </property>
   </widget>
   <widget class="QLabel" name="no_label2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="text">
     <string>Sayı 2: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_line">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>39</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_line2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>79</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_toplama">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Toplama</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cikarma">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Çıkarma</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_carpma">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Çarpma</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_bolme">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Bölme</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_sonuc">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sonuç: </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>669</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
